ton_client/src/debot/tests.rs

```diff
@@ -568,9 +700,32 @@ async fn test_debot_interface_call() {
         { "choice": EXIT_CHOICE, "inputs": [], "outputs": [] }
     ]);
     TestBrowser::execute(
-        client.clone(),
-        debot_addr.clone(),
-        keys.clone(),
-        serde_json::from_value(steps).unwrap()
+        client,
+        debot_addr,
+        keys,
+        serde_json::from_value(steps).unwrap(),
+        vec![],
+    ).await;
+}
+
+#[tokio::test(core_threads = 2)]
+async fn test_debot_4() {

```

Comment:
Just one test? I think new debot interfaces require more complex testing

Reply:
Debot smart contract validates data returned from interface calls. It throws an exception if data is invalid.
Will add checks on client side: check that target contract was deployed, check for return args of target's get-method.